pkg/operator/starter.go

```diff
@@ -213,10 +213,15 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		operatorClient,
 		controllerContext.EventRecorder,
 	).
-		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos-extended.yaml"}, notOnSingleReplicaTopology, nil).
+		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos-extended.yaml", "assets/alerts/cpu-utilization.yaml"}, notOnSingleReplicaTopology, nil).
 		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos.yaml"}, never, nil). // TODO remove in 4.13
 		AddKubeInformers(kubeInformersForNamespaces)
 
+	err = workloadpartitioning.CreateSNOAlert(ctx, dynamicClient, infrastructure.Status, controllerContext.EventRecorder)
```

Comment:
why hasn't it be implemented as a controller ?
/cc @dgrisonnet (since you reviewed the original pr)

Reply:
I think it should be a controller because it would allow you to reconcile any potential changes to that resource and it would prevent the operator from exiting on error. Does it make sense ?

Reply:
I see two issues here. The first is that the alert can be changed, and then the controller should repair such a resource. The second is that we shouldn’t send direct requests to the server but rather read data from the cache.
I’m sorry but after an internal discussion with the team, we decided that we should revert the original PR.

/cc @tkashem @vrutkovs @benluddy

Reply:
Yeah, definitely not the best idea.

`StaticResourceController`, which should be used there,  can template manifests and watches the resulting reosurces

Reply:
@vrutkovs I don't see how the [StaticResourceController](https://github.com/openshift/cluster-kube-apiserver-operator/blob/d597184dccf293101d38e543587d40af99aca52a/vendor/github.com/openshift/library-go/pkg/operator/staticresourcecontroller/static_resource_controller.go#L96) could be used to change the resource. It would be nice if you could share an example.

Reply:
Easiest way is to [replace the variable in the asset](https://github.com/openshift/cluster-csi-snapshot-controller-operator/blob/66048040bb40915f0903a0e7de0bbc6f4b4faef5/pkg/operator/starter.go#L165) and

Reply:
given that the code freeze is imminent soon, my recommendation is to revert this PR and then we open a new PR that achieves the same goal using a controller pattern